src/port1.0/portbuild.tcl

```diff
@@ -171,7 +171,13 @@ proc portbuild::build_getjobsarg {args} {
     global use_parallel_build
     if {![tbool use_parallel_build]} {
         ui_debug "port disallows a parallel build"
-        return ""
+        # Ninja defaults to enabling parallelism if -jX is omitted
+        # so need to explicitly pass -j1 to disable this
+        if { [string match "*ninja*" [option build.cmd]] } {
+            return " -j1"
+        } else {
+            return ""
+        }
```

Comment:
Due to the below change, we'll already be using `-j1` for make/scons sometimes, so why not do it here as well?
```suggestion
        return " -j1"
```

Reply:
The final change is after the check

```
   if {![exists build.jobs] || \
            !([string match "*make*" [option build.cmd]] || \
              [string match "*ninja*" [option build.cmd]] || \
              [string match "*scons*" [option build.cmd]])} {
        return ""
    }
```

so we know the build system is one of three, that support -j. I didn't do it here as I presumed there might be other build systems not supporting -j.